bigquery/google/cloud/bigquery/client.py

```diff
@@ -1256,6 +1257,9 @@ def list_rows(self, table, selected_fields=None, max_results=None,
         :param start_index: (Optional) The zero-based index of the starting
                            row to read.
 
+        :type page_size: int
+        :param page_size: (Optional) The maximum number of items to return.
```

Comment:
This docstring is very misleading. Unless `page_size` is just an alias for `max_results`?

I would hope that in setting the page size, I could still fetch all pages via the iterator.

Reply:
From this, [paging-results](https://cloud.google.com/bigquery/docs/paging-results)

it looks like the default value is 50.  Please let me know if that's not the case.